README.md

```diff
@@ -1371,6 +1402,16 @@ Note that the private key **must** be base64 encoded, e.g. by running
 $ base64 -i private.key
 ```
 
+To run tests with OAuth credentials, add the fllowing to `.env`:
+
+```
+OAUTH_E2E_CLIENT_ID=<CLIENT ID>
+OAUTH_E2E_CLIENT_SECRET=<CLIENT SECRET>
+OAUTH_E2E_TA_ID=<TECHNICAL ACCOUNT ID>
+OAUTH_E2E_TA_EMAIL=<TECHNICAL ACCOUNT EMAIL>
+OAUTH_E2E_IMS_ORG_ID=<ORG ID>
```

Comment:
Actually, the tests can run independently. So there's a chance JWT and OAuth integration can be in different orgs
I think it's subjective, because if we keep org ID same then we're basically implying we need both integrations to belong to same org when actually its independent

Reply:
IMO, If both integrations can exist within the same org, it implies the possibility of independent integrations as well. Conversely in different orgs, it suggests that simultaneous integrations within same org may not be feasible.